src/containers/FeedPage/PostCard/index.js

```diff
@@ -0,0 +1,88 @@
+  import React from 'react';
+  import Card from '@material-ui/core/Card';
+  import CardContent from '@material-ui/core/CardContent';
+  import CardActions from '@material-ui/core/CardActions';
+  import IconButton from '@material-ui/core/IconButton';
+  import Typography from '@material-ui/core/Typography';
+  import ArrowUpward from "@material-ui/icons/ArrowUpward";
+  import ArrowDownward from "@material-ui/icons/ArrowDownward";
+  import Button from "@material-ui/core/Button";
+  import { connect } from "react-redux"
+  import styled from "styled-components"
+  import { setPostId, votePost } from "../../../actions/votePosts"
+  import { push } from "connected-react-router"
+  import { routes } from "../../Router/"
+
+  const PostCardContainer = styled.div`
+    margin: 10px;
+  `
+
+
+  class PostCard extends React.Component {
+    
+    constructor(props) {
+      super(props);
+    }
```

Comment:
Quando não declaramos o estado (`this.state`) não precisa declarar o construtor.